core/include/optee_rpc_cmd.h

```diff
@@ -148,6 +148,18 @@
  */
 #define OPTEE_RPC_CMD_BENCH_REG		20
 
+/*
+ * Issue master requests (read and write operations) to an I2C chip.
+ *
+ * [in]     value[0].a	    Transfer mode (check enum rpc_i2c_mode for modes).
+ * [in]     value[0].b	    The I2C bus (a.k.a adapter).
+ * [in]     value[0].c	    The I2C chip (a.k.a address).
+ * [in]     value[1].a	    The I2C master control flags (ie, 10 bit address).
+ * [in/out] memref[2]	    Buffer used for data transfers.
+ * [out]    value[3].a	    Number of bytes transferred by the REE.
+ */
```

Comment:
I'm missing the defines
``` C
#define OPTEE_MSG_RPC_CMD_I2C_TRANSFER_RD 0
#define OPTEE_MSG_RPC_CMD_I2C_TRANSFER_WR 1
```
found in the kernel patch. We'd like to minimize the diff with the corresponding description in the kernel driver.

The master control flags supported by this ABI needs to be specified here.

It would be nice the specify which bits are unused and should be zero in `value[0].b`, `value[0].c` and `value[1].a`. That will make it easier if we one day need to extend this ABI.

Reply:
yes I was wondering about this as well, I noticed too it was missing on the kernel patch and I couldnt figure out if it was intentional or not. So I guess not. Sure will add.